packages/components/src/public/QueryModel/ChartPanel.tsx

```diff
@@ -54,10 +61,10 @@ export const ChartPanel: FC<Props> = memo(({ actions, model, api = DEFAULT_API_W
         (successMsg?: string) => {
             setShowEditModal(false);
             if (successMsg) {
-                createNotification({ message: successMsg, alertClass: 'success' });
+                _createNotification({ message: successMsg, alertClass: 'success' });
```

Comment:
Won't this be undefined for LKS? Should it be `_createNotification?.(...)`?

Reply:
Shoot, yes. I missed that part. I'll fix. Thanks.